CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Changelog
+## 4.07
```

Comment:
I would call this "Unreleased", since we will probably wait for all these changes before cutting a new release

Reply:
fixed.